FirebaseMLModelDownloader/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# Unreleased
+- [changed] Migrated from the Firelog to Clearcut telemetry logging endpoint. (#11837)
```

Comment:
This won't change any public functionality so I think we should skip the changelog entry.

Reply:
SGTM. Reverted this file.